src/commandrouting.cpp

```diff
@@ -230,31 +257,7 @@ int CommandRouter::processSerialStream() {
     return 0;
   }
 
-  // This could likely be replaced by strtok_r, but I really failed
-  // at using it on the teensy 4.0
-  // And as such, I'm simply ignoring using it.
-  int i;
-  i = 0;
-  argc = 0;
-  while (i < bytes_read){
-    char c;
-    c = input_buffer[i];
-    if (c == '\0')
-      break;
-     if (input_buffer[i] == ' ') {
-       input_buffer[i] = '\0';
-       i++;
-       continue;
-    }
-    argv[argc] = &input_buffer[i];
-    argc++;
-    while(i < bytes_read) {
-        i += 1;
-        c = input_buffer[i];
-        if (c == ' ' || c  == '\0')
-            break;
-    }
-  }
+  parseInputBuffer(input_buffer, bytes_read, argv, argc);
```

Comment:
is this right? argv is not declared anywhere in CommandRouter::processSerialStream

Reply:
They should be part of the class. I think